Library/Formula/gtkextra.rb

```diff
@@ -11,7 +11,7 @@ class Gtkextra < Formula
   end
 
   depends_on "gtk+"
-  depends_on "pkg-config" => :run
+  depends_on "pkg-config" => :build
```

Comment:
It seems `pkg-config` was only used in test. So we can remove this line completely?

(Also, please try not to ping people in the commit message. I got too many notifications from that. Thanks.)